packages/angular/tsconfig.ngc.json

```diff
@@ -0,0 +1,26 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+// This tsconfig is used when building @sentry/angular with the Angular
+// compiler and `ng-packagr`. It configures a production build conforming
+// to the Angular Package Format (APF).
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "target": "es2015",
+    "lib": [
+      "dom",
+      "es2018"
```

Comment:
I'm curious about the mismatch of versions here. Does the angular compiler provide polyfills for es2016-es2018 stuff?

Reply:
Also here this is what was originally set by the Angular CLI.
I think what this basically allows us to do is to theoretically use es2018 but `ngc` compiles the sources to es2015. So yes, I guess it provides polyfills for >es2015 stuff.